server/lib/chart.js

```diff
@@ -120,73 +121,110 @@ function barChart(params) {
     });
   }
 
+  const dataParameter = {
+    labels : chart.labels,
+    datasets : chart.data,
+  };
+
+  const options = {
+    chart_canvas_id : canvasId,
+    chart_data : dataParameter,
+    chart_x_axis_label : xAxesLabelString,
```

Comment:
Is there someway to specify that the values should be rendered as currency values?

It would be really nice to render this in the enterprise currency of FC:
![image](https://user-images.githubusercontent.com/896472/109466047-63d52580-7a69-11eb-8c9b-79968934ac9a.png)

It is a bit hard to read as it is.

Reply:
it gave me a headache today 😄, you will see my progress in the commit below